src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.sendletter.controllers;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.sendletter.model.Letter;
+import uk.gov.hmcts.reform.sendletter.services.LetterService;
+
+import java.util.Map;
+
+@RestController
+@RequestMapping(
+    path = "letters",
+    produces = {MediaType.APPLICATION_JSON_VALUE}
+)
+public class SendLetterController {
+
+    private final LetterService letterService;
+
+    public SendLetterController(LetterService letterService) {
+        this.letterService = letterService;
+    }
+
+    @PostMapping(consumes = {MediaType.APPLICATION_JSON_VALUE})
+    @ApiOperation(value = "Send letter to print and post service")
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "Successfully sent letter"),
+    })
+    public void sendLetter(@RequestBody Map<String, String> addressDetails) {
```

Comment:
is void 200?

Reply:
No that will change.Currently kept it void but will update it later